package.xml

```diff
@@ -36,6 +36,7 @@
   <build_depend>google-mock</build_depend>
   <build_depend>python-sphinx</build_depend>
 
+  <depend>async_grpc</depend>
```

Comment:
I don't think the Cartographer ros package should depend on async_grpc or grpc. The default should be no grpc (at least for ros users).

Reply:
I don't think this specifies a mandatory dependency; its only purpose is if you have `async_grpc` in your Catkin workspace, this lets Catkin know it should build that first - AFAIK.  We can test if that is the case, and if it doesn't break things when you're building without grpc, can we leave it here?

Reply:
Sorry for the delayed review.
I believe that ROS' package build server fail to find this (async_grpc is just our github project, not a ROS or linux package), and in my eyes this package.xml should support the official ros package of cartographer (where we don't use async_grpc.

Reply:
We're a bit stuck on this. There's no way do specify an optional dependency in Catkin's package.xml; on the other hand, if we don't specify there, Catkin won't know that it should build `async_grpc` first if we _are_ building cartographer_grpc. Not sure how to resolve this.

Reply:
In my eyes, the priority here is to support the official ROS release, which should not depend on async_grpc. Luckily, this config also works for most users that build cartographer_ros themselves.

Reply:
Okay, I removed the `<depend>` line. The worst thing about that can happend is that the user may have to run `catkin build async_grpc` (or `catkin_make_isolated --pkg async_grpc`) manually.